### ISSUE TITLE
Errors in scneario analysis with theoretical dataset

### ISSUE THREAD

**Issue by lisphilar** [lisphilar/covid19-sir#155]
## Summary
When use theoretical data, we know the parameter values without estimation.  It is the best way to apply parameter values with `Scenario.add()`, but this failed with errors.

## Codes and outputs:
```Python
import covsirphy as cs
# Instance to save population values
population_data = cs.PopulationData(filename=None)
# Set tau value and start date of records
example_data = cs.ExampleData(tau=1440, start_date="01Jan2020")
# Preset of SIR-F parameters
preset_dict = cs.SIRF.EXAMPLE["param_dict"]
# Create dataset from 01Jan2020 to 31Jan2020
area = {"country": "Theoretical"}
example_data.add(cs.SIRF, step_n=30, **area)
# Register population value
population_data.update(cs.SIRF.EXAMPLE["population"], **area)
# Create Scenario class
snl = cs.Scenario(example_data, population_data, tau=1440, **area)
# Set 0th phase from 02Jan2020 to 31Jan2020 with preset parameter values
snl.clear(include_past=True)
snl.add(end_date="31Jan2020", model=cs.SIRF, **preset_dict)
# Add main scenario
snl.add(end_date="31Dec2020", name="Main")
# Add lockdown scenario
rho_lock = snl.get("rho", phase="0th") / 2
snl.add(end_date="31Dec2020", name="Lockdown", rho=rho_lock)
# Add medicine scenario
kappa_med = snl.get("kappa", phase="0th") / 2
sigma_med = snl.get("sigma", phase="0th") * 2
snl.add(end_date="31Dec2020", name="Medicine", kappa=kappa_med, sigma=sigma_med)
# Add vaccine scenario
snl.add(end_date="31Dec2020", name="Vaccine", model=cs.SIRFV, omega=0.001)
# Summarize
snl.summary()
# Compare scenarios
snl.describe(y0_dict={"Vaccinated": 0})
```
This causes KeyError in creation of `PopulationData` instance, IndexError in `Scenario.add()` and KeyError in `Scenario.describe()` when model was change in the future phases.

## Environment
- CovsirPhy version: 2.6.0-alpha.fix.120
- Python version; 3.8
- Installation: pipenv
- OS: WSL

**Comment by lisphilar** [comment]
Parameter name "alpha1" must be change to "alpha1 [-]" as the same as that of SIR-F model.

**Comment by lisphilar** [comment]
Fixed in version 2.6.0-beta.fix.120.155
